features/consume.feature

```diff
@@ -1,7 +1,8 @@
 Feature: Consuming AMQP messages
 
 Scenario: Consuming a text message
-    Given The queue 'test_1' contains the text message "Puzzle is great"
+    Given The queue 'test_1' is empty
```

Comment:
C'est bizarre cet ajour : "Etant donné que la queue est vide et que la queue contient 1 message" C'est contradictoire ?
Je loupe qqch ?

Reply:
En fait la queue n'est pas purgée au début, donc si il restait des messages avant les tests (par exemple, en cas de plantage d'un test précédent), le test fait un faux positif.

C'est pourquoi j'ai ajouté la purge de chaque queue avant de commencer les traiter.

Reply:
Yes j'en déduis ça mais du coup le texte Gherkin est faux. 
Essaie plutôt un "Given ... contains **only** the text message ..." et rajoute la purge dans l'implem PHP

Reply:
GIven c'est l'état de départ, tu peux pas y faire un enchaînement temporel d'actions

Reply:
Bien reçu ! Je fais la modif !

Reply:
fixed avec [0e7b17f](https://github.com/puzzle-org/amqp/pull/50/commits/0e7b17f28f60c32bc310552d40dbf8ce2ad871ab) 
J'ai du refaire une correction sur la llib http